dart/lib/flat_buffers.dart

```diff
@@ -444,7 +448,7 @@ class Builder {
     _maxAlign = 1;
     _tail = 0;
     _currentVTable = null;
-    _vTables.clear();
+    _vTables.length = 0;
```

Comment:
I think this change should be made separately. Clear is the same as this.

Reply:
Another instance of a change improvement that slipped in from the fork. Coming from some talks with @mraleph (this commit: https://github.com/objectbox/objectbox-dart/commit/a72970591a582b060d943a162a5f536de6ac8df7#diff-594a84a4fd72af27284e6f59d689b04f9f1f8cfe9dbcdfb512669d16d0f702b0) - I was also a bit confused why setting length was chosen instead of clear but didn't ask at the time. Maybe this mention will prompt a helpful comment from @mraleph? Either way, I'm fine with rolling back to clear() if you want.

Reply:
Maybe it avoids a method invocation? Clear does this internally. Not really a big deal either way, but it might be good to separate out some of these performance related fixes into a separate patch. Not a big deal either way though, this shouldn't hurt anything.

Reply:
I don't actually remember why I made this change. Most likely `clear()` was not inlined, can't really find any explanation why I would do it otherwise.

Reply:
Would keep it this way if you don't mind. I can create a follow-up PR with the remaining perf improvments